client/components/product-card/README.md

```diff
@@ -160,18 +155,9 @@ export default class extends React.Component {
 	render() {
 		return (
 			<ProductCard
-				title={
-					<Fragment>
-						Jetpack Backup <strong>Daily</strong>
-					</Fragment>
-				}
+				title="Jetpack Backup Daily"
```

Comment:
Are we omitting the `<strong>` on purpose here?

Reply:
Yes, it's just an example file so for the sake of brevity I removed the `<strong>` tag.

Reply:
👍 :shipit: